wain-ast/src/lib.rs

```diff
@@ -66,23 +66,22 @@ pub enum ValType {
 }
 
 impl ValType {
-    pub fn bytes(self: ValType) -> usize {
+    #[must_use]
```

Comment:
What is the intention of adding this attribute?

Reply:
The `must_use` attribute emits a warning if a return value is not used. For example one could write:

```rust
ValType::F32.bytes();
```

This statement has no effect and is most likely a programming mistake.

https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-must_use-attribute
https://github.com/rust-lang/rfcs/blob/master/text/1940-must-use-functions.md#motivation

Reply:
ok, looks good to use it here. I think It'd be better to have some criteria to use the attribute in the future.